src/signals/incident/components/IncidentForm/index.js

```diff
@@ -52,12 +51,10 @@ class IncidentForm extends React.Component {
     this.form.meta.incident = this.props.incidentContainer.incident;
     this.form.meta.submitting = this.state.submitting;
     if (this.state.loading !== prevState.loading && !this.state.loading && this.state.next) {
```

Comment:
that is not the same: it only needs to proceed when: loading has been changed AND the loading is false both. then the form should be submitted

state management like this will not be needed anymore when we make this a functional component with useState